docs/source/miscellaneous_environment_variables.rst

```diff
@@ -8,7 +8,10 @@ Miscellaneous Environment Variables
   * - Variable
     - Description
   * - ``TORCH_FORCE_WEIGHTS_ONLY_LOAD``
-    - If set to [``1``, ``y``, ``yes``, ``true``], the torch.load will use ``weight_only=True``. For more documentation on this, see :func:`torch.load`.
+    - If set to [``1``, ``y``, ``yes``, ``true``], the torch.load will use ``weights_only=True``. For more documentation on this, see :func:`torch.load`.
```

Comment:
nit: since this is different from the other one, I think it's worth explicitly mentioning that this one will change the value even if the user explicitly provided False.